### ISSUE TITLE
`tags_extra_files` throws an exception

### ISSUE THREAD

**Issue by SaltyAimbOtter** [squidfunk/mkdocs-material#4114]
### Contribution guidelines

- [X] I've read the [contribution guidelines](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### I've found a bug and checked that ...

- [X] ... the problem doesn't occur with the `mkdocs` or `readthedocs` themes
- [X] ... the problem persists when all overrides are removed, i.e. `custom_dir`, `extra_javascript` and `extra_css`
- [X] ... the documentation does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem

### Description

The new `tags_extra_files` feature seems to not work at all. An exception is thrown whenever this feature is used. The "old" `tags_file` still works, though.

### Expected behaviour

Tag indexes are rendered on two pages.

### Actual behaviour

This [stacktrace](https://github.com/squidfunk/mkdocs-material/files/9086662/stacktrace.txt) is thrown.

### Steps to reproduce

1. `git clone https://github.com/SaltyAimbOtter/material-mkdocs-tags-bug.git`
2. `mkdocs serve`

### Package versions

- Python: `3.10.0`
- MkDocs: `1.3.0`
- Material: `8.3.9+insiders.4.20.1`

### Configuration

```yaml
site_name: My Docs
theme:
  name: material

plugins:
  - tags:
      tags_extra_files:
        index.md:
          - docs2
        anotherTagsIndex.md:
          - docs1

extra:
  tags:
    docs1: docs1
    docs2: docs2


markdown_extensions:
  - meta
```

### System information

- Operating system: Windows 10
- Browser: Firefox 102

**Comment by squidfunk** [comment]
Thanks for reporting. The `tags_extra_files` setting relies on `tags_file` being present, meaning, it will not work without a `tags_file`. The reason is the resolution of links. I'll check whether we can make it work without a `tags_file`, or emit a warning when `tags_file` is not defined.

**Comment by squidfunk** [comment]
Fixed in squidfunk/mkdocs-material-insiders@c8551a261.

**Comment by SaltyAimbOtter** [comment]
Hey Martin,

sorry for the delay. I briefly tested your fix with my demo repo and it worked. However, it did not in my actual project. 
Now I finally got around to dig deeper. 

It seems like navigatation sections don't work with several tags indexes. All tags are shown on each index subset, no matter what is configured.
I adjusted my [example repository](https://github.com/SaltyAimbOtter/material-mkdocs-tags-bug) so you can easily test.

**Comment by squidfunk** [comment]
Thanks, I'll look into it again as soon as I find some time.

**Comment by squidfunk** [comment]
I'm not able to reproduce what you described. Works perfectly for me:

![screenshot-localhost-8000-Tutorials-tagsIndex-1658057289439](https://user-images.githubusercontent.com/932156/179395969-6acbb256-de78-4751-ad46-94c3901c99b5.png)

![screenshot-localhost-8000-Documentation-anotherTagsIndex-1658057299197](https://user-images.githubusercontent.com/932156/179395966-55dfa487-80fa-47c6-af74-3ec62cb45795.png)

**Comment by squidfunk** [comment]
Released as part of 8.3.9+insiders-4.21.0.

**Comment by SaltyAimbOtter** [comment]
Interesting, multiple members from our team can reproduce it. 
We will have another look when the vacation period is over.

Thank you for your help so far!
Enjoy your vacation! 🌞
